torch/csrc/jit/script/module.h

```diff
@@ -189,70 +198,9 @@ struct NamedParameter {
   std::unique_ptr<at::Tensor> parameter;
 };
 
-// simple ordered dict used only in Module
-// contains only the minimum necessary functionality for Module
-template<typename T>
-struct OrderedDict {
-  OrderedDict(const char * what)
-  : what(what) {}
-  // note: slight difference from python here.
-  // we do not allow for insertion of an already existing value,
-  // because we not allow allow methods or submodules to be updated
-  // once created
-  T& insert(const std::string& name,  T&& value) {
-    if(index_.count(name) != 0) {
-      std::stringstream ss;
-      ss << "module " << what << "'" << name << "' already defined.";
```

Comment:
What happens to this error message in the new version? I don't see the same string in the new code. I am concerned that this means there is a regression in the user-friendliness of the API.

Reply:
I'll put it back

Reply:
I've put it back